docs/sources/getting-started/getting-started.md

```diff
@@ -1,58 +1,59 @@
 +++
-title = "Getting started"
+title = "Getting started with Grafana"
 description = "Guide for getting started with Grafana"
 keywords = ["grafana", "intro", "guide", "started"]
 type = "docs"
 aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
 [menu.docs]
-name = "Getting started"
-identifier = "getting_started_guide"
+identifier = "getting_started-grafana"
 parent = "guides"
 weight = 200
 +++
 
-# Getting started
+# Getting Started with Grafana
 
-This guide will help you get started and acquainted with Grafana. To learn more about Grafana in general, refer to [What is Grafana?]({{< relref "what-is-grafana.md" >}}).
+This topic will help you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "what-is-grafana.md" >}}).
 
-## Install Grafana
+## Step 1: Install Grafana
 
-This step varies according to your computer operating system. Refer to the instructions for your OS in the [Installation]({{< relref "../installation/_index.md" >}}) section for instructions.
+Grafana can be installed on many different operating systems. For a list of minimum hardware and software requirements, as well as instructions on installing Grafana, refer to [Install Grafana]({{< relref "../installation/_index.md" >}}).
 
-## Log in for the first time 
+## Step 2: Log in
 
-1. Open your web browser and go to http://localhost:3000/. `3000` is the default HTTP port that Grafana listens to if you haven’t configured a different port.
-1. On the login page, type `admin` for the username and password.
-1. Change your password. 
+To login to Grafana for the first time:
 
-> **Note:** We strongly encourage you to follow Grafana best practices and change the default administrator password. Don't forget to record your credentials!
+1. Open your web browser and go to http://localhost:3000/. The default HTTP port that Grafana listens to is `3000` unless you have configured a different port.
+1. On the login page, enter `admin` for username and password.
+1. Click **Log In**.
+1. If login is successful, you will see a prompt to change the password.
```

Comment:
```suggestion
 If login is successful, then you will see a prompt to change the password.
```

Reply:
This is not a step. The user is not doing anything.

Reply:
Fixed, no longer a step.